src/fides/api/service/connectors/sql_connector.py

```diff
@@ -115,6 +124,18 @@ def default_cursor_result_to_rows(results: LegacyCursorResult) -> List[Row]:
             rows.append({col[0]: row_tuple[count] for count, col in enumerate(columns)})
         return rows
 
+    @staticmethod
+    def get_namespace_meta(db: Session, dataset: str) -> Optional[Dict[str, Any]]:
```

Comment:
This is the way we pull in the namespace info into the connector. I thought about caching this for a slight performance improvement but decided against a premature optimization, keeping it simple for now

Reply:
no this is really solid, i like it 👍